content/docs/introduction/glossary.md

```diff
@@ -102,9 +110,9 @@ A remote write endpoint is what Prometheus talks to when doing a remote write.
 
 ### Silence
 
-A silence in the Alertmanager prevents alerts with labels matching the silence from
+A silence in the Alertmanager prevents alerts, with labels matching the silence, from
 being included in notifications.
 
 ### Target
 
-One application, server or endpoint that Prometheus is scraping.
+A target is the definition of a scrape. For example, what labels to apply, any authentication required to connect, or other information that defines how the scrape will occur.
```

Comment:
definition of a thing to scrape

Reply:
This is where I think this gets clumsy IMHO and why I find it hard to puzzle out instances, targets, and jobs. :)